art/attacks/inference/attribute_inference/black_box.py

```diff
@@ -173,3 +173,122 @@ def _check_params(self) -> None:
             raise ValueError("Attack feature must be either an integer or a slice object.")
         if isinstance(self.attack_feature, int) and self.attack_feature < 0:
             raise ValueError("Attack feature index must be positive.")
+
+
+class AttributeInferenceBaseline:
```

Comment:
I think we should move this class into a separate file `baseline.py`, if possible, to follow the pattern of the module.